### PR TITLE
Don't log LDAP password when server is not available

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: lib/private/Log.php [hunk 1]
```diff
@@ -82,22 +82,28 @@ class Log implements ILogger {
...
-		//LoginController
+		// LoginController
 		'tryLogin',
 		'confirmPassword',
+
+		// LDAP
+		'bind',
+		'areCredentialsValid',
+		'invokeLDAPMethod',
+		'checkPasswordNoLogging',
```
Comment (blizzz) [inline lib/private/Log.php]: that's part of server's user manager (lib/private/User/Manager.php)

**Review by blizzz — COMMENTED** [review]
One thing to avoid confusion, otherwise good.

The CI, however, looks  🐟y

File: lib/private/Log.php (hunk 1 above)
Comment (nickvergessen) [reply lib/private/Log.php]: moved

**Review by blizzz — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Backport is in #6396

**Pull request merged**